src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String paragraph = System.lineSeparator();
+        StringBuilder builder = new StringBuilder("Report for period ")
+                .append(dateFrom).append(" - ")
+                .append(dateTo);
+
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
```

Comment:
Please, make your formatter a constant (`private` `static` `final` and class level)